__tests__/app.test.js

```diff
@@ -157,3 +156,43 @@ describe('"GET /api/articles/:article_id/comments', () => {
   });
 });
 
+describe('"POST /api/articles/:article_id/comments add a comment for an article.', () => {
+  test("responds with 201 status code with an array of comments object should have comment_id,votes,created_at,author,body,article_id", () => {
+    const newComments = {
+      username: "butter_bridge",
+      body: "This is latestest comments"
+    };
+    return request(app)
+      .post("/api/articles/7/comments")
+      .send(newComments)
+      .expect(201)
+      .then((res) => {
+        const { comment } = res.body;
+        expect(comment).toHaveProperty("comment_id");
+        expect(comment.body).toBe(newComments.body);
+        expect(comment.author).toBe(newComments.username);
+        expect(comment.article_id).toBe(7);
+        expect(comment.votes).toBe(0);
+        expect(comment.created_at).toBeDefined();
+      });
+  });
+
+test('should return an error if required fields are missing or invalid username',()=>{
```

Comment:
i would change this description to just be fields missing and I would do a separate test for invalid username!